tcms/core/tests/test_ajax.py

```diff
@@ -33,6 +42,45 @@ def test_empty_string_is_converted_to_bool(self):
         response = self.client.get(url)
         self.assertEqual(200, response.status_code)
 
+    def test_with_ulli_format(self):
+        """ When a request comes with info_type=categories, and format=ulli for given product_id,
+            we expect to receive all categories for that product, rendered as HTML unordered list
+        """
+
+        url = '%s?info_type=categories&product_id=%s&format=ulli' % (reverse('ajax-info'),
+                                                                     self.product.pk)
+
+        response = self.client.get(url)
+
+        self.assertContains(response, '<li>' + self.category_one.name + '</li>')
+        self.assertContains(response, '<li>' + self.category_two.name + '</li>')
+
+    def test_with_json_format(self):
```

Comment:
I don't think this format is ever used but we should keep JSON instead of the HTML. You'd better open an issue for that.